src/bijectors/named_bijector.jl

```diff
@@ -55,146 +57,52 @@ names_to_bijectors(b::NamedBijector) = b.bs
     return :($(exprs...), )
 end
 
-@generated function inverse(b::NamedBijector{names}) where {names}
-    return :(NamedBijector(($([:($n = inverse(b.bs.$n)) for n in names]...), )))
-end
-
-@generated function logabsdetjac(b::NamedBijector{names}, x::NamedTuple) where {names}
+@generated function logabsdetjac(b::NamedTransform{names}, x::NamedTuple) where {names}
     exprs = [:(logabsdetjac(b.bs.$n, x.$n)) for n in names]
     return :(+($(exprs...)))
 end
```

Comment:
Maybe just
```julia
function logabsdetjac(t::NamedTransform{names}, x::NamedTuple{names}) where {names}
    return sum(map(logabsdetjac, t.bs, x))
end
```
(and possibly, if really needed, another method that reorders `x`)?

Reply:
Though I agree with you, I think we should defer reducing usage of `@generated` to a separate PR. I made an issue: https://github.com/TuringLang/Bijectors.jl/issues/243